src/test/java/ResourceTest.java

```diff
@@ -1,70 +1,83 @@
-    Status mockStatus = mock(Status.class);
-    public void setUp(){
-        MockitoAnnotations.initMocks(this);
-    }
-    //@ClassRule
-   //public static final ResourceTestRule RESOURCES = ResourceTestRule.builder().addResource(mock(TwitterAppResource.class)).build();
-
-    @Test
-    //POST Rest Service
-    public void testTweetLength(){
-        String shortTweet = "this is a short tweet";
-
-        try{
-        when(mockTwitter.updateStatus(shortTweet)).thenReturn(mockStatus);
-        mockResource.postTweet(shortTweet);
-        List<Status> statuses = mockTwitter.getUserTimeline();
-        mockStatus = statuses.get(0);
-        assertEquals(shortTweet, mockStatus.getText());
-        }
-        catch(Exception e){
-
-        }
-    }
-
-    @Test
-    //POST Rest Service
-    public void testLongTweet(){
-        String longTweet = "This tweet is longer than the 280 limit...................................................................................................................................................................................................................................................";
-        try{
-            when(mockTwitter.updateStatus(longTweet)).thenReturn(mockStatus);
-            mockResource.postTweet(longTweet);
-            List<Status> statuses = mockTwitter.getUserTimeline();
-            mockStatus = statuses.get(0);
-            assertFalse(longTweet.equals(mockStatus.getText()));
-        }
-        catch(Exception e){
-
-        }
-    }
-}
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import twitter4j.ResponseList;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitterapp.src.TwitterAppResource;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+public class ResourceTest {
+    TwitterAppResource resource;
+
+    @Mock
+    Twitter mockTwitter = mock(Twitter.class);
+    Response r = mock(Response.class);
+
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+        resource = new TwitterAppResource(mockTwitter);
+    }
+
+    @Test
+    public void testTweetLength(){
+        String shortTweet = "this is a short tweet part 6";
+        r = resource.postTweet(shortTweet);
+        assertEquals(200, r.getStatus());
+        assertEquals("Tweet("+shortTweet+") has been posted",r.getEntity().toString());
+    }
+
+
+    @Test
+    public void testLongTweet(){
+        String longTweet = "This tweet is longer than the 280 limit...................................................................................................................................................................................................................................................";
```

Comment:
you can use stringutils to generate a string of length 'x' instead of hardcoding one